packages/near-membrane-base/src/connector.ts

```diff
@@ -22,10 +22,14 @@ export function createBlueConnector(globalObject: typeof globalThis): Connector
     if (typeof globalObject !== 'object' || globalObject === null) {
         throw new TypeErrorCtor('Missing globalObject.');
     }
-    let createHooksCallback = globalThisToBlueCreateHooksCallbackMap.get(globalObject);
+    const { eval: evaluator } = globalObject;
```

Comment:
Two comments:

1. I'm surprised that this code is here in base rather than in dom since this only applies to iframes.
2. I believe we are safe here because the code running in the sandbox can't affect the value of window.eval on the actual iframe, only on its own virtualized version of it, but still I'm worry that we use the dot notation here... anyhow, since there are not unforgeables, I guest we have no other choice. This is the first time I wish to have unforgeables jajaja